maquetado/scss/partials/_tech.scss

```diff
@@ -6,30 +6,25 @@ $border-radius: 2%;
 
 .tech-box {
   align-items: center;
-  background-color: $color-green;
-  border-radius: $border-radius;
   display: flex;
   flex-flow: column;
   font-size: $text-medium;
-  margin: 20px;
   padding: 30px;
-  position: relative;
-  width: 200px;
-  
+  width: 250px;
+}
+
+.tech-box-feature {
```

Comment:
I'd recommend using a CSS selector instead of creating a new class. This can help you cope with items removal. 😬